openpype/hosts/nuke/plugins/load/load_clip.py

```diff
@@ -220,8 +220,21 @@ def _representation_with_hash_in_frame(self, representation):
             dict: altered representation data
         """
         representation = deepcopy(representation)
-        frame = representation["context"]["frame"]
-        representation["context"]["frame"] = "#" * len(str(frame))
+        context = representation["context"]
+        template = representation["data"]["template"]
+        frame = context["frame"]
+        hashed_frame = "#" * len(str(frame))
+
+        if (
+            "{originalBasename}" in template
+            and "frame" in context
+        ):
```

Comment:
This would crash on single frame representation
```suggestion
        if (
            "{originalBasename}" in template
            and "frame" in context
        ):
            frame = context["frame"]
            hashed_frame = "#" * len(str(frame))
```